admin_manual/installation/examples/lets_encrypt/register.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+# export makes the variable available for all subprocesses
+
+# Assumes that mydom.tld www.mydom.tld and sub.mydom.tld are the domains that you want a certificate for
+export DOMAINS="-d mydom.tld -d www.mydom.tld -d sub.mydom.tld"
+
+$( which certbot ) certonly --config /etc/letsencrypt/cli.ini "$DOMAINS" # --dry-run
```

Comment:
- There was a reason why I did not use ``which``. ``man which``: "It does this by searching the PATH for executable files matching the names of the  arguments."
For installing certbot from git in ``/opt/letsencrypt`` or any other location not defined in ``echo $PATH``, ``which`` does not return a result and the shell script will fail. In addition git uses certbot-auto, ppa uses certbot. 
- Proposing to revert back in all bash scripts  and return to the previous way accessing certbot/certbot-auto.

Reply:
I'll have to read over certbot/certbot-auto, but wrt `which`, after re-reading the man page, I totally get what you're saying, and see my mistake.